airbyte-integrations/connectors/destination-teradata/metadata.yaml

```diff
@@ -2,19 +2,24 @@ data:
   connectorSubtype: database
   connectorType: destination
   definitionId: 58e6f9da-904e-11ed-a1eb-0242ac120002
-  dockerImageTag: 0.1.2
+  dockerImageTag: 0.1.3
   dockerRepository: airbyte/destination-teradata
   githubIssueLabel: destination-teradata
   icon: teradata.svg
   license: MIT
   name: Teradata Vantage
+  normalizationConfig:
+    normalizationIntegrationType: null
+    normalizationRepository: null
+    normalizationTag: null
```

Comment:
I would remove this.  I think your testing env was problematic

Reply:
I tried to setup airbyte on multiple instances but observed same behavior. I updated actor_definition_version supportsDBT as true and can able to see custom transformation UI only. normalizationConfig fields are kept empty. So with this I am assuming normalizationConfig is not required to specified with null

Reply:
Shell I remove these and recommit the file?

Reply:
please remove them